File: matlab/lib/+Ice/ConnectionIdleException.m
% ConnectionIdleException   Summary of ConnectionIdleException
%
% This exception indicates that a connection was aborted by the idle check.

% Copyright (c) ZeroC, Inc. All rights reserved.

classdef ConnectionIdleException < Ice.LocalException
    methods
        function obj = ConnectionIdleException(errID, msg)
            if nargin == 0 || isempty(errID)
                errID = 'Ice:ConnectionIdleException';
            end
            if nargin < 2 || isempty(msg)
                msg = 'Ice.ConnectionIdleException';
            end
            obj = obj@Ice.LocalException(errID, msg);
        end
    end
end

File: java/src/Ice/src/main/java/com/zeroc/Ice/ConnectionIdleException.java
// Copyright (c) ZeroC, Inc.

package com.zeroc.Ice;

/** This exception indicates that a connection was aborted by the idle check. */
public class ConnectionIdleException extends LocalException {
  public ConnectionIdleException() {
    super();
  }

  public ConnectionIdleException(Throwable cause) {
    super(cause);
  }

  public String ice_id() {
    return "::Ice::ConnectIdleException";
  }

  private static final long serialVersionUID = -1271371420507272518L;
}
